deepspeed/ops/transformer/inference/op_binding/base.py

```diff
@@ -0,0 +1,9 @@
+import torch
+import deepspeed
+
+
+class BaseOp(torch.nn.Module):
+    def __init__(self, builder="InferenceBuilder"):
```

Comment:
It looks like every child op is taking in the config right now, so I think it makes sense to maybe make this explicit with the base class taking in the config as well.